### DISCUSSION TITLE
Different theme per server

### DISCUSSION THREAD

**Discussion by thickey256** [isontheline/pro.webssh.net#1169] [Ideas]
Would it be possible to set the theme per server profile?  I often have a session open on my local machine and also on my server at the same time.

If I could set the theme to be different for each, then I can easily tell based on the background color or whatever which server I am on.  and less likely to break something important if I get it wrong.

**Comment by isontheline** [comment]
Hello @thickey256 👋

Of course you can setup a different theme for each server : edit your connection > terminal settings > theme >> specific 

Could you make it a try?
